measure_extinction/extdata.py

```diff
@@ -815,6 +839,36 @@ def read(self, ext_filename):
 
                 self.columns[curkey] = (float(pheader.get(curkey)), tunc)
 
+        # get the fitted model if available
+        if "MODEXT" in extnames:
+            self.model["waves"] = hdulist["MODEXT"].data["MOD_WAVE"]
+            self.model["exts"] = hdulist["MODEXT"].data["MOD_EXT"]
+            self.model["residuals"] = hdulist["MODEXT"].data["RESIDUAL"]
+            self.model["params"] = []
+            paramkeys = [
+                "AMPLITUD",
+                "X_0",
+                "ALPHA",
+                "SCALE",
+                "X_O",
+                "GAMMA_O",
+                "ASYM",
+                "AV",
+            ]
```

Comment:
This will only work for the NIR model.  But not for the MIR for FM90 models.  As the parameters are saved, maybe this could be determined from the file itself?  Or maybe the model parameters should be stored another way that is easier to read?  Maybe a table as another extension?